### PR TITLE
Add dependency management

### PR DESCRIPTION
**Pull request by andyatlant**
1) add dependency management root pom
2) add jUnit
3) add dependency chat common to chat client

### CODE REVIEW

File: chat-client/pom.xml [hunk 1]
```diff
@@ -14,4 +14,15 @@
...
 
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.chat.skype</groupId>
+            <artifactId>chat-common</artifactId>
+            <version>${project.version}</version>
```
Comment (vbogdanovsky) [inline chat-client/pom.xml:25]: please add chat-common to chat-server too

Comment (andyatlant) [inline chat-client/pom.xml:25]: I will do it in the next branch

Comment (andyatlant) [inline chat-client/pom.xml:25]: Done

**Comment by vbogdanovsky** [comment]
:+1:

**Pull request merged**